CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.12.0] - 2019-12-17
```

Comment:
```suggestion
## [0.12.0] - 2020-01-02
```

Reply:
We'll release 0.12 whenever we finish merging everything. The last PR should update this